hbw/app/javascript/packs/hbw/components/form/submit_select_button.js.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useContext } from 'react';
+import cx from 'classnames';
+import TranslationContext from 'shared/context/translation';
+import compose from 'shared/utils/compose';
+import { withConditions, withCallbacks } from 'shared/hoc';
+
+const HBWSubmitSelectButton = ({
+  params,
+  error,
+  onClick,
+  task,
+  disabled: buttonDisabled,
+  hidden: buttonHidden,
+  submitSelectDisabled
+}) => {
+  const { translateBP } = useContext(TranslationContext);
+  const disabled = submitSelectDisabled || buttonDisabled;
+
+  const cssClass = cx(params.css_class, { disabled });
+  const faClass = cx(params.fa_class, { disabled });
+
+  const label = translateBP(`${task.process_key}.${task.key}.${params.name}`, {}, params.label);
+
+  return !buttonHidden && <button
+    type="submit"
+    className={cssClass}
+    title={params.title}
+    onClick={onClick}
+    disabled={error || disabled}
+  >
+    <i className={faClass} />
+    {label}
+  </button>;
+};
+
+export default compose(withCallbacks, withConditions)(HBWSubmitSelectButton);
```

Comment:
Why do you need `withCallbacks` here?

Reply:
Because we use trigger here 
https://github.com/latera/homs/blob/master/hbw/app/javascript/packs/hbw/components/shared/hoc/conditions.js#L80
That is taken from here
https://github.com/latera/homs/blob/master/hbw/app/javascript/packs/hbw/components/shared/hoc/callbacks.tsx#L40